config/Dockerfiles/rpmbuild/Dockerfile

```diff
@@ -53,5 +53,6 @@ COPY rpmbuild-test.sh /tmp/rpmbuild-test.sh
 
 # Run the build script
 ENTRYPOINT ["bash", "/tmp/rpmbuild-test.sh"]
+#
```

Comment:
Is this needed or a typo? Just wondering if I will need to add this to future containers, not trying to nit pick

Reply:
this is a dummy change I need to validate the change I am doing surrounding the promotion of images AFTER a merge